creator-node/src/userReqMiddleware.js

```diff
@@ -0,0 +1,59 @@
+const { userReqLimiter } = require('./reqLimiter')
+const { getSPInfo, verifyRequesterIsValidSP } = require('./components/replicaSet/URSMRegistrationComponentService')
+
+/*
+if ((route == 'batch_clock' or route == 'individual clock')) {
+  try {
+    isValidSP()
+    next()
+  } catch (E) {
+    // probably not a valid SP. this is ok
+  }
+}
+rateLimit()
+*/
+
+// For the clock fetching routes, if the requester if from a valid SP, do not enforce
+// a rate limit as we do not want to rate limit SPs checking other SPs for clock values.
+// The clock fetching routes are used in issuing potential reconfig ops.
```

Comment:
they're used in Reconfig and sync ops, maybe just say they are used programmatically by CNodes to monitor each other or smth